### DISCUSSION TITLE
Discrepancies in data in Tutorial

### DISCUSSION THREAD

**Discussion by tanuj0518** [satijalab/seurat#4067] [General]
I started the Seurat Guided Cluster Tutorial but I began to have issues replicated the graphs and figures made after the "Preform Linear Dimensional  Reduction" section. I copy and pasted line for line from the tutorial and had all the same figures and values leading up to that section; however, when I run:
"print(pbmc[["pca"]], dims = 1:5, nfeatures = 5)" 

I get :
PC_ 1 
Positive:  FTL, FTH1, CST3, AIF1, S100A4 
Negative:  MALAT1, IL32, LTB, IL7R, CCL5 
PC_ 2 
Positive:  S100A8, S100A9, LYZ, CD14, LGALS2 
Negative:  B2M, NKG7, GZMA, CTSW, PRF1 
PC_ 3 
Positive:  CD74, HLA-DRA, HLA-DPB1, HLA-DQA1, HLA-DQB1 
Negative:  PPBP, GNG11, SPARC, PF4, AP001189.4 
PC_ 4 
Positive:  CD74, HLA-DQA1, HLA-DQB1, HLA-DQA2, HLA-DRA 
Negative:  FCGR3A, NKG7, FTL, RP11-290F20.3, S100A4 
PC_ 5 
Positive:  FCGR3A, RP11-290F20.3, MS4A7, MALAT1, CKB 
Negative:  NKG7, LYZ, S100A9, S100A8, GNLY 

Instead of: 
## Negative:  LTB, IL7R, CKB, VIM, MS4A7

And after that point, the PCA graphs are incorrect and the clusters have a different count and different shapes. If you could help that would be great.

**Comment by tanuj0518** [comment]
Figured it out. Thank you!

**Reply by dgj24** [reply]
Hi Tanuj, can you please explain how you fixed this error? 
I am currently getting: 
PC_ 1
Positive: MALAT1, LTB, IL32, IL7R, CD2
Negative: CST3, TYROBP, LST1, AIF1, FTL 

It seems like the same genes are being identified, but I don't understand why the expression differences are switched.
